src/sequelize.js

```diff
@@ -960,7 +963,12 @@ class Sequelize {
       ...options
     };
 
-    await this.query('SELECT 1+1 AS result', options);
+    let sql = 'SELECT 1+1 AS result';
```

Comment:
since you need to customize this query, can you add `AbstractQueryGenerator#authTestQuery`? (OracleQueryGenerator can the customize it)

Reply:
Added abstract method authTestQuery